kolibri/plugins/learn/assets/src/views/content-card-group-carousel/index.vue

```diff
@@ -102,12 +95,17 @@
         // flag marks holds the index (in contents array, prop) of first item in carousel
         contentSetStart: 0,
         // flag that marks when the slide animation will be going start at left
-        leftToRight: false,
+        panBackwards: false,
```

Comment:
Direction stopped mattering

Reply:
nice. Yes in general "right" and "left" are not general enough for variable names